packages/host/package.json

```diff
@@ -88,6 +88,7 @@
   "devDependencies": {
     "@babel/core": "^7.26.10",
     "@babel/types": "^7.27.0",
+    "@types/babel__core": "^7.20.5",
```

Comment:
What was this fixing?

Reply:
https://github.com/callstackincubator/react-native-node-api/pull/204#issuecomment-3116203736